manifests/j/Jabra/Direct/5.5.39795/Jabra.Direct.locale.en-US.yaml

```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Jabra.Direct
+PackageVersion: 5.5.39795
+PackageLocale: en-US
+Publisher: Jabra
+PackageName: Direct
```

Comment:
```suggestion
Publisher: GN Audio A/S
PackageName: Jabra Direct
```

Reply:
Doesn't this sorta mean that the package identifier should then also become GnAudioAs.JabraDirect? (Wow that is one ugly identifier!)